docs/src/main/asciidoc/datasource.adoc

```diff
@@ -229,16 +233,34 @@ As previously stated, most of the time, this automatic resolution will suit you
 you won't need to configure the driver.
 ====
 
-Note that you can still define a specific driver if you need to (for instance for using the OpenTracing driver)
-or if you want to use a database for which we don't have a built-in JDBC driver extension.
+[[other-databases]]
+==== Use a database with no built-in extension or with a different driver
+
+You can use a specific driver if you need to (for instance for using the OpenTracing driver)
+or if you want to use a database for which Quarkus does not have a built-in JDBC driver extension.
 
-Defining a specific JDBC driver is as simple as:
+Without an extension, the driver will work fine in any Quarkus app running in JVM mode.
+It is unlikely that it will work when compiling your application to a native executable though.
+If you plan to make a native executable, we recommend you use the existing JDBC Quarkus extensions (or contribute one for your driver).
+
+Here is how you would use the OpenTracing driver.
 
 [source, properties]
 ----
 quarkus.datasource.jdbc.driver=io.opentracing.contrib.jdbc.TracingDriver
 ----
 
+Here is how you would define access to a database with no built-in support (in JVM mode)
```

Comment:
```suggestion
Here is how you would define access to an Oracle database with no built-in support (in JVM mode)
```

Reply:
Not convinced. Keeping it as is

Reply:
Okay, I though we could explicit the target database. I guess it does not add that much considering we have the driver right after in config properties.